src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,124 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd?: (movie: Movie) => void;
```

Comment:
```suggestion
  onAdd: (movie: Movie) => void;
```
Just FYI - it shouldn't be optional